### ISSUE TITLE
Logspout on Kubernetes crashing without information

### ISSUE THREAD

**Issue by thecodeassassin** [gliderlabs/logspout#298]
Hi we wanted to use logspout to send node-level docker logs to papertrail. However, after a certain (around 10 hours) of perfectly logging everything papertrail stops receiving any data. When i inspect the nodes they are still running but logspout seems to be inactive. We are using the following yml.

@ardigo has reported the same issue in https://github.com/gliderlabs/logspout/issues/202#issuecomment-293889848

```
apiVersion: extensions/v1beta1
kind: DaemonSet
metadata:
  name: logspout
  labels:
    tier: monitoring
    app: logspout
    version: v1
spec:
  template:
    metadata:
      labels:
        name: logspout
    spec:
      hostPID: true
      hostIPC: true
      hostNetwork: true
      containers:
        - resources:
            requests:
              cpu: 0.2
              memory: 200Mi
          securityContext:
            privileged: true
          env:
            - name: SYSLOG_TAG
              value: '{{ index .Container.Config.Labels "io.kubernetes.pod.namespace" }}[{{ index .Container.Config.Labels "io.kubernetes.container.name" }}]'
            - name: SYSLOG_HOSTNAME
              value: '{{ index .Container.Config.Labels "io.kubernetes.pod.name" }}-{{ index .Container.Config.Labels "io.kubernetes.pod.namespace" }}'
            - name: ROUTE_URIS
              value: syslog+tls://logs5.papertrailapp.com:XXXXX
            - name: INACTIVITY_TIMEOUT
              value: 1m
          image: gliderlabs/logspout
          name: logspout
          volumeMounts:
            - name: log
              mountPath: /var/run/docker.sock
      volumes:
        - name: log
          hostPath:
              path: /var/run/docker.sock

```

**Comment by michaelshobbs** [comment]
this should be fixed in v3.2.1. give it a go and let us know if you're still seeing the issue

dup #268

**Comment by thecodeassassin** [comment]
@michaelshobbs trying v3.2.1 if it fixes the problem
